metaci/build/autoscaling.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import subprocess
+
+import django_rq
+import requests
+from cumulusci.core.utils import import_global
+from django.conf import settings
+from rq import Worker
+from rq.registry import StartedJobRegistry
+
+logger = logging.getLogger(__name__)
+
+
+class Autoscaler(object):
```

Comment:
i suspect this base class may want to expose a structured log line for (current_workers, demand, delta) or whatever, something you can extract the metric on heroku of how often you have dynos stilling with no work waiting to scale down